published_topic_names:
  landoltc_alert : /vision/landoltc_alert
subscribed_topic_names:
  predator_topic_name : /vision/predator_landoltc_output  
